search/migrations/0001_initial.py

```diff
@@ -1,47 +1,39 @@
-# Generated by Django 2.2.4 on 2019-10-14 23:12
+# Generated by Django 2.2.6 on 2019-10-24 04:04
 
+import datetime
 from django.db import migrations, models
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
-    dependencies = []
+    dependencies = [
+    ]
 
     operations = [
         migrations.CreateModel(
-            name="CraigslistLocation",
+            name='CraigslistLocation',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
I hope it would not be a problem if applying the changes in the primary key and foreign key. Though I might need to delete this migration, and run migrate again and create a new one.

Reply:
well your fix will go in first so don't worry about it, I'll handle fixing the conflict next week.